src/js/View/DashboardView.deprecated.js

```diff
@@ -219,13 +219,13 @@ App.View.Dashboard = Backbone.View.extend({
       });
       var wasteLviWidgetModel = new Backbone.Model({
         'className': 'lvigauge',
-        'var_id':'waste.ilv.indicator',
+        'var_id': 'waste.ilv.indicator',
         // 'timeinstant':timeinstant
-        'date': scope === 'guadalajara' ? new Date(2017,5,1) : new Date(2016,7,1),
+        'date': scope === 'guadalajara' ? new Date(2017, 9, 1) : new Date(2016, 7, 1),
```

Comment:
It's not a good practice to include "MAGIC STRINGS" into the source code. It seems problematic also to include "vertical specific logic" into the core part. There is not a better option to do this?

Reply:
You are totally right @mrutid. In this case this is a legacy code before Urbo, there is a pending refactor for it since we open the source code but it's not done yet and that's why this file is called 'deprecated'.
I've updated this legacy code because it's needed right now, but our goal is to migrate all this vertical code to the 'Urbo panel architecture'. Thank you for your advice!